### PR TITLE
chore: add colorclaim resource

### LINKED ISSUE #938: [Add]: COLOR RESOURCE 💡
### Site URL

https://vanschneider.com/colors

### Description of the above provided site

- ColorClaim is simple. All I do is collect my favorite color combinations on one big page. Usually these combinations are very subtle, one main and one  accent color.
- It Collects & combine unique colors for your future projects.

### Category

Frontend

### Sub Category

colors

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by 0xabdulkhalid**
## Changes proposed

- Added `ColorClaim` resource to Frontend -> colors category.

## Screenshots

Screenshot 2023-06-12 at 12-04-24 LinksHub A hub of ready-to-use tech resources

### CODE REVIEW

**Comment by rupali-codes** [comment]
@0xabdulkhalid can you please add _closes #issuenumber_ in the description of this PR?

File: database/frontend/colors.json [hunk 1]
```diff
@@ -124,5 +124,12 @@
     "url": "https://www.khroma.co/",
     "category": "frontend",
     "subcategory": "colors"
+  },
+  {
+    "name": "ColorClaim",
+    "description": "ColorClaim is a color library that helps designers, photographers, and others create beautiful and effective color palettes.",
```
Comment (rupali-codes) [inline database/frontend/colors.json:130]:
```suggestion
    "description": "ColorClaim is a library of colors that helps developers, designers, photographers, and others to create beautiful and effective color palettes.",
```

**Comment by 0xabdulkhalid** [comment]
Requested changes has been done @rupali-codes, you can verify them!

**Review by rupali-codes — APPROVED** [review]
lgtm

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Pull request merged**